test/Page/ExpressivePageTest.php

```diff
@@ -0,0 +1,308 @@
+<?php
+
+namespace ZendTest\Navigation\Page;
+
+use PHPUnit\Framework\TestCase;
+use Zend\Diactoros\ServerRequest;
+use Zend\Expressive\Router\Exception\RuntimeException as RouterException;
+use Zend\Expressive\Router\Route;
+use Zend\Expressive\Router\RouteResult;
+use Zend\Expressive\Router\ZendRouter;
+use Zend\Navigation\Exception\InvalidArgumentException;
+use Zend\Navigation\Page\ExpressivePage;
+
+class ExpressivePageTest extends TestCase
+{
+    /**
+     * @var \Zend\Expressive\Router\RouterInterface
+     */
+    private $router;
+
+    /**
+     * @var \Zend\Expressive\Router\Route
+     */
+    private $route;
+
+    /**
+     * @var \Zend\Expressive\Router\RouteResult
+     */
+    private $routeResult;
+
+    protected function setUp()
+    {
+        $this->route = new Route('/foo', 'foo', ['GET'], 'foo');
+
+        $this->router = new ZendRouter();
+        $this->router->addRoute($this->route);
+
+        $this->routeResult = $this->router->match(
+            $request = new ServerRequest(
+                ['REQUEST_METHOD' => 'GET'], [], '/foo', 'GET'
+            )
+        );
+    }
+
+    public function testGetHref()
+    {
+        $page = new ExpressivePage(
+            [
```

Comment:
We can move it to line above, and then content can have one less indent and closing bracket could be with `);`